client/src/lib/components/sandbox/play/Arena.svelte

```diff
@@ -2,9 +2,15 @@
 	import { truncateMinaPublicKey } from '$lib/utils';
 	import { afterUpdate, onMount } from 'svelte';
 	import * as Utils from './utils';
+	import { MinaArenaClient } from '$lib/mina-arena-graphql-client/MinaArenaClient';
+
+	import { drawMovementPhase } from './phases/movement-phase';
 
 	export let game: Game;
+	let currentPlayerMinaPubKey: string = game.currentPhase?.gamePlayer.player.minaPublicKey || '';
```

Comment:
Eventually the public key will come from the client, but this is fine for the purposes of moving things along.

In fact, there is a file, `client/src/lib/stores/sandbox/playerStore.ts` which defines our 2 dummy players - in the future it would probably expose the actual identity of the player instead.  Stores are a svelte concept to manage global state on the frontend.  The idea is to give us access to a piece of state without having to either pass it around as a param, or query it in the component.

You can either `import { dummyPlayer1, dummyPlayer2 } from '$lib/stores/sandbox/playerStore';`, which is just the 2 constants, or even better, `import { player } from '$lib/stores/sandbox/playerStore';`, which you can use to read and write the address of the `current player` of the game, for use everywhere else.